### DISCUSSION TITLE
Play 3 files every 10 minutes

### DISCUSSION THREAD

**Discussion by 918483** [savonet/liquidsoap#2750] [General]
Hello everyone how do I run 3 files from a folder every 10 minutes?

**Comment by cdgraff** [comment]
Take a look here: https://www.liquidsoap.info/doc-2.1.2/cookbook.html

Search Scheduling, and Playlist, combined them can do what you need.

If need more support I recommend the SLACK channel

**Comment by 918483** [comment]
in case I would like to pull these 3 files from a folder every 10 minutes or 15 minutes, following the order of the folder or mixing, would you have any ideas for me?

**Comment by 918483** [comment]
????
